libpysal/weights/weights.py

```diff
@@ -1197,15 +1190,14 @@ def set_transform(self, value="B"):
                 # variance stabilizing
                 weights = {}
                 q = {}
-                k = self.cardinalities
                 s = {}
                 Q = 0.0
                 self.weights = self.transformations["O"]
                 for i in self.weights:
                     wijs = self.weights[i]
                     q[i] = math.sqrt(sum([wij * wij for wij in wijs]))
                     s[i] = [wij / q[i] for wij in wijs]
-                    Q += sum([si for si in s[i]])
+                    Q += sum([si for si in s[i]])  # noqa C416
```

Comment:
```suggestion
                    Q += sum(s[i])
```
It should look like this no? or
```suggestion
                    Q += sum(list(s[I]))
```

need to be tested but both should be equal imho

Reply:
I was quite proud when I completely re-wrote this using that nice groupby we have in `graph` :)